x/bep3/genesis.go

```diff
@@ -20,6 +20,11 @@ func InitGenesis(ctx sdk.Context, keeper Keeper, supplyKeeper types.SupplyKeeper
 		panic(fmt.Sprintf("failed to validate %s genesis state: %s", ModuleName, err))
 	}
 
+	// only set the previous block time if it's different than default
```

Comment:
should we set an empty time (`time.Time{}` or `time.Unix(0, 0)`) if it's the same as the default?

Reply:
I guess I don't understand why do we need the condition here. A comment would help clarify that

Reply:
👍  this was unnecessary